qemu/tests/cfg/hotplug_virtio_mem.cfg

```diff
@@ -0,0 +1,32 @@
+- hotplug_virtio_mem:
+    only Linux
+    only Host_RHEL.m9
+    only RHEL.9
```

Comment:
Suggest `no` old release version

Reply:
Changed now, thanks!

Reply:
I mean remove `only RHEL.9`, `only Host_RHEL.m9` and use `no ***`.  consider the new version, RHEL10, RHEL 11 and so on.

Reply:
Changed now, virtio-mem support is from rhel9 onwards so for the future rhel10 and rhel11 should be supported, thanks!